libs/blocks/chart/chart.js

```diff
@@ -33,9 +33,10 @@ const parseValue = (value) => (Number.isInteger(Number(value)) ? parseInt(value,
 
 export function processDataset(data) {
   const dataset = {};
-
-  // Remove group and unit from headers
-  const headers = Object.keys(data[0]).filter((header) => !['unit', 'group', 'color'].includes(header.toLowerCase()));
+  const extraHeaders = ['unit', 'group', 'color', 'unit2'];
```

Comment:
Previously multiple units were separated by dashes, though this was only used for scatter chat, alternatively only the first row is used so the second row could be used instead of using a new column for one entry.

Reply:
@Brandon32 I mistakenly left `unit2` in. I'll remove it. I started by adding a second column and then realized we used the dash last time.